incubator/factorio/templates/NOTES.txt

```diff
@@ -0,0 +1,38 @@
+Getting Started:
```

Comment:
Can you remove this, it's made redundant since Helm will output `NOTES:` as the header before this.